db/migrate/20230206143214_create_webhooks.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CreateWebhooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :webhooks, id: :uuid do |t|
+      t.references :organization, index: true
+
+      t.uuid :object_id, null: false
+      t.string :object_type, null: false
+
+      t.integer :status, default: 0
+      t.integer :retries, default: 0
```

Comment:
```suggestion
      t.integer :status, default: 0, null: false
      t.integer :retries, default: 0, null: false
```

Reply:
since we have a default is it useful?

Reply:
I think yes, default is set only when creating the row, it does not prevent to set the value to null afterwards